ruby/cross_service_examples/item-tracker/src/db_wrapper.rb

```diff
@@ -0,0 +1,182 @@
+# frozen_string_literal: true
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+require "logger"
+require "sequel"
+require "multi_json"
+require_relative "report"
+
+class RDSResourceError < Exception
+end
+
+class RDSClientError < Exception
+end
+
+# Issues commands directly to the Amazon RDS Data Service, including SQL statements.
+class DBWrapper
```

Comment:
Is there a better name for this?

Reply:
I'm sure. Do you have any suggestions?

Reply:
Your class creates and gets work items, so maybe something domain related like work_item.rb